vtex-apps/checkout-summary
# Feature/installments block

#### What problem is this solving?

Creating block summary-installments to show installments related information. Similar to product-installments block from product-price app.
This feature depends on this PR.

#### How to test it?

Go to [Workspace](https://addnewblockcheckoutsummary--tokstokio.myvtex.com/) add a product to cart and hover over minicart icon. Inspect SummaryContextProvider element with React DevTools to confirm paymentData being passed as prop.

#### Screenshots or example usage:

![image](https://user-images.githubusercontent.com/81425175/113144128-4ed3e980-9203-11eb-8b8a-f787ac8b1ac8.png)

#### How does this PR make you feel? [:link:](http://giphy.com/)

![giphy](https://user-images.githubusercontent.com/81425175/113144331-80e54b80-9203-11eb-9463-e2c075d9a7ee.gif)
